src/ovp_pipeline/unified_pipeline_enhanced.py

```
@@ -250,23 +250,27 @@ def _check_api_key() -> tuple[bool, str]:
     return True, "OK"
 
 
-def init_env_file() -> int:
+def init_env_file(vault_dir: Path | str | None = None) -> int:
     """初始化 .env 文件（交互式）"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)